quantus/helpers/pytorch_model.py

```diff
@@ -87,3 +87,31 @@ def get_random_layer_generator(self, order: str = "top_down", seed: int = 42):
             torch.manual_seed(seed=seed + 1)
             module[1].reset_parameters()
             yield module[0], random_layer_model
+
+    def get_hidden_layers_outputs(self, x: np.ndarray,
+                                  layer_names: Optional[List[str]] = None,
+                                  layer_indices: Optional[List[int]] = None,
+                                  ) -> np.ndarray:
+        # skip last layer, and module defined by subclassing API
```

Comment:
Missing docstring, please go over all functions and add where necessary.

Reply:
there's one in base class https://github.com/aaarrti/Quantus/blob/relative-stability/quantus/helpers/model_interface.py#L53-L68.